site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -39,7 +39,7 @@ The port of the SMTP server that FusionAuth will use.
 Additional Email Configuration in a properties file formatted String.
 
 [field]#{base_field_name}.emailConfiguration.security# [type]#[String]# [since]#Available since 1.8.0#::
-The type of security protocol FusionAuth will use when connecting to the SMTP server. The possible values are:
+The type of security protocol FusionAuth will use when connecting to the SMTP server. The following are valid values:
```

Comment:
This is fine, however we have a lot of existing usages of this wording to describe the available values for a parameter. 

The most common usage we have is `The possible values are:`, `The following are valid values:`, and I think we have a few other outliers as well that have slightly different wording. 

If you think this is a better way to describe parameter values, we could do a find on all similar uses and see if it makes sense to change them to `The following are valid values:`.

Reply:
No, I think I changed it because I thought I was being consistent. I'll revert this change.